src/kitchen/components/lines/Lines.stories.tsx

```diff
@@ -3,7 +3,7 @@ import type { Story } from '../../../@types/storybook-emotion-10-fixes';
 import { asPlayground, asChromaticStory } from '../../../lib/story-intents';
 
 export default {
-	title: 'Editorial/src-ed-lines/Lines',
+	title: 'Kitchen/source-kitchen-react/Lines',
```

Comment:
Is this the best place to put the stories? Alternatively should it be:

* Source/src-kitchen-react/Lines
* Kitchen/Lines
* Something else?

Reply:
Kitchen/Lines as there will only be one package